spec/controllers/websites_controller_spec.rb

```diff
@@ -1,17 +1,100 @@
 require "rails_helper"
 require "spec_helper"
 
-describe WebsitesController, "creating a new website" do
+describe WebsitesController do
+  let(:website) { double(Website).as_null_object }
 
-	it "should redirect to new website path" do 
-  		redirect_to new_website_path
-	end
+  describe "GET new" do
+    login_user
+    it "responds successfully" do
+      get :new
+      expect(response).to be_success
+    end
+    it "renders the new template" do
+      get :new
+      expect(response).to render_template("new")
+    end
+  end
 
-	it "should create variables named h1_title, paragraph and final_content" do
-		 expect(@h1_title).to eq(nil)
-		 expect(@paragraph).to eq(nil)
-		 expect(@final_content).to eq(nil)
-		
-	end
+  describe "POST create" do
+    login_user
```

Comment:
You can pull this at top of the spec, because you have it in every describe block.